openapi/openapi.yaml

```diff
@@ -554,6 +555,18 @@ components:
                 version_id:
                   type: string
                   example: "29"
+            runtime_environment:
+              type: object
+              description: Runtime environment information.
+              properties:
+                cpu_type:
+                  type: dict
+                cpu_info:
+                  type: dict
+                cpu_features:
+                  type: dict
+                platform:
+                  type: dict
```

Comment:
I thought there will be reported `operating_system` (and dested `name`, `version`) and hardware information in complience with Thoth's RuntimeEnvironment (so it can be directly used during syncs with conf2models method).

These cpu* things would be also good to have, but it can be declared as `object` (no need to enumerate each) so its not validated using openapi.

Reply:
Actually yes, I made confusion here :angel: